src/ai/backend/manager/registry.py

```diff
@@ -1581,13 +1582,8 @@ async def finalize_running(
                 ),
             }
             self._kernel_actual_allocated_resources[kernel_id] = actual_allocs
-            kernel_did_update = await KernelRow.update_kernel(
-                self.db, kernel_id, new_status, update_data=update_data
-            )
-            if not kernel_did_update:
```

Comment:
Is there a specific reason lifting up this condition?

Reply:
hm, I don't remember exactly. I reverted this part. thanks!